src/components/Helpers/SpanWithLinebreaks.tsx

```diff
@@ -8,6 +9,7 @@ const Span = styled.span`
   white-space: pre-line;
 `
 
-const SpanWithLinebreaks: React.FC<Props> = ({ s }) => <Span>{ s }</Span>
+const createMarkup = (s: string = "") =>  ({ __html: s })
+const SpanWithLinebreaks: React.FC<Props> = ({ s }) => <Span dangerouslySetInnerHTML={ createMarkup( ReplaceURLs( s ) ) } />
```

Comment:
This could be shortened to `<Span dangerouslySetInnerHTML={ { __html: replaceURLs(s) } } />`

Reply:
Done